Formula/b/beancount-language-server.rb

```diff
@@ -0,0 +1,19 @@
+class BeancountLanguageServer < Formula
+  desc "Language server for beancount files"
+  homepage "https://github.com/polarmutex/beancount-language-server"
+  url "https://github.com/polarmutex/beancount-language-server/archive/refs/tags/v1.3.4.tar.gz"
+  sha256 "d56f8d2d30f471b5f5011ffff9b55da9fecaec13f34a5f8f2ec0d6aa28cd3ded"
+  license "MIT"
+  head "https://github.com/polarmutex/beancount-language-server.git", branch: "main"
+
+  depends_on "rust" => :build
+  depends_on "beancount"
```

Comment:
Installing `beancount` does not sound strictly necessary according to https://github.com/polarmutex/beancount-language-server#requirements, so I think we could defer to the user to install this.